### ISSUE TITLE
DIfference between multitarget_KN_grounded_CN dataset and hsgencnkp_test dataset

### ISSUE THREAD

**Issue by wadhwasahil** [yilingchung/Towards_KN_CN_Generation#3]
Hi,

Thanks for the repo. I have a few questions.

1. Where is the training data in the multitarget_KN_grounded_CN dataset https://github.com/marcoguerini/CONAN/tree/master/multitarget_KN_grounded_CN?
2. Inside the folder, https://github.com/yilingchung/Towards_KN_CN_Generation/tree/main/KN_retriever/data/KN_CONAN_final_data, how are the datasets generated and how is v2 different from the corresponding versions like [hscnkp_test_v2.txt]?

Sahil

**Comment by yilingchung** [comment]
Hi Sahil,

I missed this before. Thanks for asking.

1. We didn't make such a split in our paper. The CNs in the dataset are gold CNs paired with gold KN retrieved and HS. You can use it for your experiments as you desire. We use it as a gold test set to evaluate our systems.

2. The datasets are extensions of CONAN dataset paired with automatically retrieved relevant knowledge sentences from Newsroom (Grusky et al., 2018) and WikiText-103  (Merity et al., 2017) using Solr and ROUGEL-F1. Please see more details in Section 5 in our paper. Apart from format and tokenization, the v2 is slightly better than the corresponding version regarding the retrieved knowledge quality.
